libfs/file.go

```diff
@@ -26,6 +27,9 @@ type File struct {
 	node     libkbfs.Node
 	readOnly bool
 	offset   int64
+
+	lockedLock sync.Mutex
```

Comment:
Confusing naming, maybe `fileLock`?

Reply:
I hear you, but I don't think it's worth a PR right now.  Next time I touch this code I'll make it happen.